ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/fees/FeeMarketConfig.java

```diff
@@ -74,4 +74,9 @@ public long getInitialBasefee() {
   public long getPerTxGaslimit() {
     return perTxGaslimit;
   }
+
+  @Override
+  public double getSlackCoefficient() {
+    return 0;
```

Comment:
Looks like we have to return the slackCoefficient here?

Reply:
Yeah nice catch.

Reply:
Fixed